src/Microsoft.AspNet.Diagnostics/Views/RuntimeInfoPage.cshtml

```diff
@@ -14,6 +14,9 @@
 
     public RuntimeInfoPageModel Model { get; set; }
 }
+@{
+    Response.ContentType = "text/html; charset=utf-8";
+}
 <!DOCTYPE html>
```

Comment:
@Eilon I now tried validating the generated page with the validator and it didn't indicate any errors (yes, there were 3 empty lines at the beginning of the document)